extension/mediator.js

```diff
@@ -61,7 +61,7 @@ class Mediator {
         if (!this.languageDetection.navigatorLanguage.includes(this.languageDetection.pageLanguage.language)) {
```

Comment:
The code inside "if" statement shouldn't be called if language pair is not supported. It triggers calls to infobar and creates unnecessary Translation object. Also we'll need to record "unsupported" metrics from mediator. It means we need to read model registry from mediator and move the check here from TranslationNotificationManager.

Reply:
| Also we'll need to record "unsupported" metrics from mediator.

Could you please elaborate on what does this mean? That we are supposed to send a ping to every time a page is not supported by the extension (i.e. all the time the user browses to a new page?)

Reply:
See this [todo](https://github.com/mozilla/firefox-translations/pull/38/files#diff-ce3e24cd36ecb87c17863aa23203ed33457bb09e419fd2dd192ba23576d24f50R88). We should always record "unsupported" metric. How the ping sending is triggered is decoupled from recording. In the current implementation, it happens on page closing. If it won't work for us, we can move this metric to a different ping and send it on timer in the future.

Reply:
Ok, so this means sending a ping to every page a user browses to any page, no? At least one ping will always be sent  if the page's lang is either supported or unsupported.

Reply:
Only when language mismatch happens. Yes, exactly one ping will be sent if the user language is different from the page language in the current implementation.